src/coreclr/jit/emit.h

```diff
@@ -250,6 +250,7 @@ struct insGroup
     unsigned int   igFuncIdx; // Which function/funclet does this belong to? (Index into Compiler::compFuncInfos array.)
     unsigned short igFlags;   // see IGF_xxx below
     unsigned short igSize;    // # of bytes of code in this group
+    insGroup*      igLoopBackEdge; // "last" back-edge that branches back to an aligned loop head.
```

Comment:
This seems potentially expensive, e.g. for MinOpts scenario where it's not used.

Should it be under `#if FEATURE_ALIGN_ALIGN`?

Reply:
Sure, makes sense.